src/backend/aws/lambda.ts

```diff
@@ -0,0 +1,36 @@
+import AWS from 'aws-sdk';
+import {
+  Opts,
+  Request,
+  Response,
+  ServiceResponse,
+  IFunctionAdapter,
+} from '../../types';
+
+class LambdaAdapter implements IFunctionAdapter {
+  private client: AWS.Lambda;
+
+  constructor(client: AWS.Lambda) {
+    this.client = client;
+  }
+
+  public async request(
+    service: ServiceResponse,
+    request: Request,
+    opts?: Opts,
+  ): Promise<Response> {
+    const params = {
+      ...opts,
+      FunctionName: service.rid,
+      Payload: JSON.stringify(request.body),
+    };
+
+    return {
+      body: 'test',
+    };
+    // const { Payload: payload, StatusCode: statusCode } = await this.client.invoke(params).promise();
```

Comment:
Looks like this should be uncommented and the above removed?

Reply:
Haha jeez, yep!!